ably/options.go

```diff
@@ -408,6 +408,10 @@ type clientOptions struct {
 	// LogHandler controls the log output of the library. This is a function to handle each line of log output.
 	// platform specific (TO3c)
 	LogHandler Logger
+
+	// InsecureAllowBasicAuthWithoutTLS permits an API key to be used even if the connection
+	// will not use TLS, something which would otherwise not be permitted for security reasons.
+	InsecureAllowBasicAuthWithoutTLS bool
```

Comment:
Can we make this assertive =>
`AllowBasicAuthWithoutTLS`

Reply:
```suggestion
	AllowBasicAuthWithoutTLS bool
```

Reply:
The point of the "insecure" prefix is to make it clear to users that this is an insecure option, it's a common approach (e.g. `insecureSkipVerify` to disable TLS verification, or `WithInsecureCredentials` to use plain text credentials with gRPC).

Reply:
Okay, if it's only let users know that it's `insecure` option, default value for the same should be `false` right?

Reply:
Seems default value of `InsecureSkipVerify` is set to `false`. It will remain the same if we interpret it as `SkipVerify`. I think prefixing with `Insurecure` shouldn't invert the underlying value

Reply:
I think prefixing with `Insurecure` shouldn't invert the underlying value

Reply:
Okay, seems by default it's set to `false`, I was confused with `WithInsecureAllowBasicAuthWithoutTLS` setting it to default value